src/main/groovy/org/gradle/android/AndroidCacheFixPlugin.groovy

```diff
@@ -31,11 +31,9 @@ class AndroidCacheFixPlugin implements Plugin<Project> {
         new AndroidJavaCompile_ProcessorListFile_Workaround(),
         new ExtractAnnotations_Source_Workaround(),
         new IncrementalTask_CombinedInput_Workaround(),
-        new StreamBasedTask_CombinedInput_Workaround(),
         new ProcessAndroidResources_MergeBlameLogFolder_Workaround(),
         new CheckManifest_Manifest_Workaround(),
-    ]
-
+    ] as List<Workaround>
```

Comment:
IntelliJ complains if this is not here.

Reply:
Interesting, I suggested removing it since "my" IntelliJ (2017.2.5) did not complain.